notification-channel-config-service-api/src/main/proto/org/hypertrace/notification/config/service/v1/notification_channel.proto

```diff
@@ -13,6 +13,13 @@ message NotificationChannelMutableData {
   string channel_name = 1;
   repeated EmailChannelConfig email_channel_config = 2;
   repeated WebhookChannelConfig webhook_channel_config = 3;
+  AwsS3BucketChannelConfig bucket_channel_config = 4;
```

Comment:
other destinations are repeated, shouldn't this one be too?

Reply:
>  because the way I am thinking about a channel is that it is owned by a team (so Ideally a team should have a single S3 source)

This seems like a logical leap - a channel is designed to aggregate destinations that are interested in a particular notification. That's not necessarily team specific, nor does a team necessarily only want the data in one s3 bucket.

I think the repeated was more for flexibility given the uncertainty. If you model a single bucket with a repeated here, we can always use that as singular up the stack (such as in the UI, for example), and then we have the option later to relax it without breaking anything. If you model a bucket with a singular here, you're forced into breaking the API or carrying tech debt to support multiple later.

Reply:
Okay, makes sense.